frontend/src/app/components/employee/modals/change-passw-modal/change-passw-modal.component.ts

```diff
@@ -27,16 +30,28 @@ export class ChangePasswComponent implements OnInit {
   ngOnInit() {
    this.userObject = this.sessionService.getUserData();
    this.id = this.userObject.id;
-  }
+  }   
+
   
   sendPassw() {
-   this.api.employee().changePassword(this.id, this.firstPass, this.secondPass).subscribe(data => {
-     this.dialogRef.close();
-   })
+
+   this.api.employee().changePassw(this.firstPassword, this.secondPassword, this.oldPassword).subscribe(data => {
+    this.dialogRef.close();
+  }, error => {
+    this.error = true;
+  })  
+
   }
   
   onCancel(): void {
     this.dialogRef.close();
   }
-   
+
+  checkPasswords(): boolean {
+    if (this.firstPassword === this. secondPassword)
```

Comment:
nem lenne ez így egyszerűbb?
```suggestion
    return this.firstPassword === this.secondPassword;
```

Reply:
kész